client/state/action-types.js

```diff
@@ -194,7 +194,12 @@ export const KEYRING_SERVICES_REQUEST_SUCCESS = 'KEYRING_SERVICES_REQUEST_SUCCES
 export const LAYOUT_FOCUS_SET = 'LAYOUT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_ACTIVATE = 'LAYOUT_NEXT_FOCUS_ACTIVATE';
+export const MEDIA_ITEMS_RECEIVE = 'MEDIA_ITEMS_RECEIVE';
```

Comment:
While working on reduxifying shortcodes (#8857) I noticed that some of the functionality requires granular information about media items receiving (for example https://github.com/Automattic/wp-calypso/pull/8857/files#diff-ac2308bd4d1d8c2075ad658ea3b46b5bR55). To support that, in addition to `MEDIA_ITEMS_RECEIVE` we'll probably have to dispatch a `MEDIA_ITEM_RECEIVE` for each single media item. You can observe the current flux actions for that - `RECEIVE_MEDIA_ITEMS` and `RECEIVE_MEDIA_ITEM`.